.github/workflows/_e2e_tests.yaml

```diff
@@ -146,6 +151,7 @@ jobs:
           E2E_TEST_RUN: ${{ matrix.test }}
           KONG_CLUSTER_VERSION: ${{ matrix.kubernetes-version }}
           TEST_KONG_CONTROLLER_IMAGE_LOAD: ${{ inputs.kic-image }}
+          TEST_KONG_KONG_IMAGE_OVERRIDE: ${{ inputs.kong-image }} # Kong images are still remote
```

Comment:
Nit: that comment might end up being helpful for some but it'd weird to have it only in 1 of those added lines.

Reply:
Dunno, style preference thing, where I tend to omit repeated comments after the first for like lines in close proximity to try and limit boilerplate, since I find the repetition distracting :shrug: